frontend/src/model/beta/images/CodeLine.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Tooltip } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import useNotification from 'src/common/Snackbar'
+
+export default function CodeLine({ line }: { line: string }) {
+  const theme = useTheme()
+  const sendNotification = useNotification()
+
+  const handleButtonClick = () => {
+    navigator.clipboard.writeText(line)
+    sendNotification({ variant: 'success', msg: 'Copied to clipboard' })
+  }
+
+  return (
+    <div
+      style={{
+        cursor: 'pointer',
+      }}
+      role='button'
+      tabIndex={0}
+      onClick={() => {
+        handleButtonClick()
+      }}
+      onKeyDown={(e) => {
+        if (e.key === 'Enter') {
```

Comment:
This should also work for spacebar. Both spacebar and enter are used to activate buttons.